docs/versionpicker.html

```diff
@@ -0,0 +1,115 @@
+<!-- Start interactive version picker -->
+<div id="installation" v-cloak>
+  <header class="row">
+    <h3>Paste this into the <code>&lt;head&gt;</code> of your HTML:</h3>
+    <button type="button" @click="copyToClipboard">
+      <transition>
+        <span v-if="copyStatus === 'success'" class="emoji">✔</span>
+        <span v-else-if="copyStatus === 'failed'" class="emoji">❌</span>
+      </transition>
+      Copy to clipboard
+    </button>
+  </header>
+
+  <transition mode="out-in">
+    <pre :key="selectedVersion.fileSnippet"><code>{{selectedVersion.fileSnippet}}</code></pre>
+  </transition>
+
+  <h3>Version options:</h3>
+  <form action="./" method="post" @submit.prevent>
+    <label>
+      <input type="radio" value="dark" checked name="theme" v-model="versionOptions.theme" />
+      Dark theme&nbsp;<span class="emoji">🌙</span>
+      <span
+        v-if="!versionOptions.isStandalone"
+        class="emoji translucent"
+        title="Selected theme may be overwritten by theme settings on user's device"
+      >
+        / ☀
+      </span>
+    </label>
+    <label>
+      <input type="radio" value="light" name="theme" v-model="versionOptions.theme" />
+      Light theme&nbsp;<span class="emoji">☀</span>
+      <span
+        v-if="!versionOptions.isStandalone"
+        class="emoji translucent"
+        title="Selected theme may be overwritten by theme settings on user's device"
+      >
+        / 🌙
+      </span>
+    </label>
+    <br /><br />
+    <label>
+      <input type="checkbox" checked v-model="versionOptions.isStandalone" />
+      Enforce theme? (ignore <code>prefers-color-scheme</code>)
+    </label>
+    <br /><br />
+    <label>
+      <input type="checkbox" v-model="versionOptions.isLegacy" />
+      Support Internet Explorer and other legacy browsers?
+    </label>
+  </form>
+
+  <h3>Version info:</h3>
+  <table>
+    <tbody>
+      <tr>
+        <th>File</th>
+        <td>{{ selectedVersion.fileName }}</td>
+      </tr>
+      <tr>
+        <th>Size (min+gzip)</th>
+        <td>{{ selectedVersion.fileSize }}KB</td>
+      </tr>
+      <tr>
+        <th>Theme</th>
+        <td>
+          <template v-if="versionOptions.isStandalone">
+            <span v-if="versionOptions.theme === 'dark'" class="emoji">🌙</span>
+            <span v-else class="emoji">☀</span>
+            {{ versionOptions.theme | capitalize }}
+          </template>
+
+          <template v-else>
+            <span
+              v-if="versionOptions.theme === 'dark'"
+              class="emoji"
+              title="Selected theme may be overwritten by theme settings on user's device"
```

Comment:
It took me a little while to figure out what the pair of 🌙 / ☀ icons meant and didn't realize that the tooltip was here to explain things. Maybe an `<abbr>` element to put an explicit underline to signal the tooltip to the user?

Reply:
I added the underline. Though not with `<abbr>` as that element uses text-decoration and we need to move the line down a few pixels, otherwise it collides with the Emoji. (and `text-underline-position: under` has atrocious browser support)
So I added a separate `.tooltip` class :)